shared/js/initApp.js

```diff
@@ -1,157 +1,113 @@
 // TODO (brent) - way too many globals
 // TODO (brent) - I wonder if we should sub-namespace dashboard
-/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga, digestManifest*/
 
-var timing = require('./timing');
-var chrome34Fix = require('./chrome34Fix');
-dashboard.project = require('./project');
+window.setupApp = function () {
+  var timing = require('./timing');
+  var chrome34Fix = require('./chrome34Fix');
```

Comment:
I would prefer to have the require's outside of the function. They all get bundled together anyways.

I also wonder if we're better off trying to limit what we're exposing as globals. Maybe instead have something like apps.setupApp and apps.initApp? We could even be more explicit about this and wrap everything in something like this:
```
window.apps = (function () {
  ...
  return {
    initApp: function () {},
    setupApp: function () {}
  }
})();
```
As I write that, it also strikes me that it's not immediately apparent from the names what setup does vs. init. I wonder if we can come up with better names, or at the very least preface these methods with a short comment explaining what they do.